src/main/java/name/valery1707/kaitai/KaitaiGenerator.java

```diff
@@ -246,11 +256,31 @@ public KaitaiGenerator noVersionCheck(boolean noVersionCheck) {
 		return this;
 	}
 
+	/**
+	 * Set no auto read mode.
+	 *
+	 * @param noAutoRead Version check mode
+	 */
+	public void setNoAutoRead(boolean noAutoRead) {
+		this.noAutoRead = noAutoRead;
+	}
+
+	/**
+	 * Set no auto read mode.
+	 *
+	 * @param noAutoRead Version check mode
+	 * @return self
+	 */
+	public KaitaiGenerator noAutoRead(boolean noAutoRead) {
+		setNoAutoRead(noAutoRead);
+		return this;
+	}
+
 	private ProcBuilder process(Logger log) {
 		ProcBuilder builder = new ProcBuilder(getKaitai().normalize().toAbsolutePath().toString())
-			.withErrorStream(new TeeOutputStream(LogWriter.logError(log), streamError))
-			.withOutputStream(new TeeOutputStream(LogWriter.logInfo(log), streamOutput))
-			.withExpectedExitStatuses(0);
+				.withErrorStream(new TeeOutputStream(LogWriter.logError(log), streamError))
+				.withOutputStream(new TeeOutputStream(LogWriter.logInfo(log), streamOutput))
+				.withExpectedExitStatuses(0);
```

Comment:
Please revert the indentation changes
```suggestion
			.withErrorStream(new TeeOutputStream(LogWriter.logError(log), streamError))
			.withOutputStream(new TeeOutputStream(LogWriter.logInfo(log), streamOutput))
			.withExpectedExitStatuses(0);
```